mailpoet/tests/integration/Cron/Workers/SendingQueue/Tasks/NewsletterTest.php

```diff
@@ -243,38 +249,46 @@ public function testItSavesNewsletterPosts() {
   }
 
   public function testItUpdatesStatusAndSetsSentAtDateOnlyForStandardAndPostNotificationNewsletters() {
-    $newsletter = $this->newsletter;
+    $newsletter = $this->newslettersRepository->findOneById($this->newsletter->id);
+    $this->assertInstanceOf(NewsletterEntity::class, $newsletter);
     $queue = new \stdClass();
     $queue->processedAt = date('Y-m-d H:i:s');
 
     // newsletter type is 'standard'
-    $newsletter->type = Newsletter::TYPE_STANDARD;
-    $newsletter->status = 'not_sent';
-    $newsletter->save();
+    $newsletter->setType(NewsletterEntity::TYPE_STANDARD);
+    $newsletter->setStatus('not_sent');
+    $this->newslettersRepository->persist($newsletter);
+    $this->newslettersRepository->flush();
     $this->newsletterTask->markNewsletterAsSent($newsletter, $queue);
-    $updatedNewsletter = Newsletter::findOne($newsletter->id);
-    assert($updatedNewsletter instanceof Newsletter);
-    expect($updatedNewsletter->status)->equals(Newsletter::STATUS_SENT);
-    expect($updatedNewsletter->sentAt)->equals($queue->processedAt);
+    $updatedNewsletter = $this->newslettersRepository->findOneById($newsletter->getId());
+    assert($updatedNewsletter instanceof NewsletterEntity);
```

Comment:
I failed to notice that some of the code that I moved around in this PR already contained calls to `assert()`. Thanks for pointing that out. I just added a new commit to remove all calls to `assert()` inside the integration tests that are modified in this PR. I left a call to `assert()` inside `SendingQueue.php` as it is not that trivial to replace and it will eventually get removed as we continue working on replacing Paris with Doctrine.